zgw_consumers/nlx.py

```diff
@@ -126,3 +128,22 @@ def post_response(
         if response_data:
             self.rewriter.backwards(response_data)
         super().post_response(pre_id, response_data)
+
+
+def get_nlx_services() -> list:
+    config = NLXConfig.get_solo()
+    if not config.outway:
+        return []
+
+    directory = config.directory_url
+    url = f"{directory}api/directory/list-services"
+
+    response = requests.get(url)
+    response.raise_for_status()
+
+    services = response.json()["services"]
+    services.sort(key=lambda x: (x["organization_name"], x["organization_name"]))
```

Comment:
why sort twice on the same key if you can just sort on a simple key?

```suggestion
    services.sort(key=lambda x: x["organization_name"])
```